### PR TITLE
Fixed the reference to mask_channel_prob in task cfg

### PR DESCRIPTION
**Pull request by ultrons**
Updated example config file for tpu to include mask parameters.
Noted currently cli bug in README

## What does this PR do?
Fixes # (3741) B.

### CODE REVIEW

File: fairseq/tasks/audio_pretraining.py [hunk 1]
```diff
@@ -140,8 +140,8 @@ class AudioPretrainingConfig(FairseqDataclass):
         },
     )
 
-    inferred_w2v_config: Optional[InferredW2vConfig] = field(
-        default=None,
```
Comment (alexeib) [inline fairseq/tasks/audio_pretraining.py:144]: do we still need to change this? can we keep it as before?

File: fairseq/tasks/audio_pretraining.py [hunk 2]
```diff
@@ -242,7 +242,7 @@ def load_dataset(self, split: str, task_cfg: FairseqDataclass = None, **kwargs):
                 **self._get_mask_precompute_kwargs(task_cfg),
             )
 
-        if self.cfg.tpu and task_cfg["mask_channel_prob"] == 0.0:
+        if self.cfg.tpu and task_cfg.inferred_w2v_config.mask_channel_prob == 0.0:
```
Comment (alexeib) [inline fairseq/tasks/audio_pretraining.py:245]: 👍

**Review by alexeib — CHANGES_REQUESTED** [review]

File: fairseq/tasks/audio_pretraining.py (hunk 1 above)
Comment (ultrons) [reply fairseq/tasks/audio_pretraining.py:144]: It was one of the intermediate changes we did when trying to solve the nested dataclass issue. The right answer is I don't know. Do you recommend to revert it.?

Comment (ultrons) [reply fairseq/tasks/audio_pretraining.py]: Changed back to orginal optional and default value.

**Pull request closed without merging**